wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1060,7 +1067,11 @@ public final class SessionManager: NSObject, SessionManagerType {
     }
 
     // Creates the user session for @c account given, calls @c completion when done.
-    private func startBackgroundSession(for account: Account, with coreDataStack: CoreDataStack) -> ZMUserSession {
+    private func startBackgroundSession(
+        for account: Account,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration,
```

Comment:
There's no need to inject it if it's retained by the session manager.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e01d45cf65dc39fe679bd1f27685ca072c192df9